ihatemoney/utils.py

```diff
@@ -12,6 +12,31 @@
 from datetime import datetime, timedelta
 
 import csv
+import requests
+
+
+class CurrencyConverter(object):
+    api_url = "https://api.exchangerate-api.com/v4/latest/USD"
+    response = []
+
+    def __init__(self):
+        self.response = requests.get(self.api_url).json()
```

Comment:
Implementing the conversion this way make it problematic when the service is not available. What happens if we can't call the service? Currently it means that the server will probably return an HTTP 500 error to the user.

One way to solve this would be to have a sort of "state" attached to the records, telling if the conversion has been done or not. If it hasn't been done, then we could try to do it later on (maybe on a subsequent operation). In the meantime, no balance will be computed, and an error message should probably be shown to the end user.